packages/logs/src/domain/logsCollection/logger/loggerCollection.ts

```diff
@@ -0,0 +1,46 @@
+import { includes, display, combine, ErrorSource, isExperimentalFeatureEnabled } from '@datadog/browser-core'
+import type { CommonContext } from '../../../rawLogsEvent.types'
+import type { LifeCycle } from '../../lifeCycle'
+import { LifeCycleEventType } from '../../lifeCycle'
+import type { Logger, LogsMessage } from '../../logger'
+import { StatusType, HandlerType } from '../../logger'
+
+export const STATUS_PRIORITIES: { [key in StatusType]: number } = {
+  [StatusType.debug]: 0,
+  [StatusType.info]: 1,
+  [StatusType.warn]: 2,
+  [StatusType.error]: 3,
+}
+
+export function startLoggerCollection(lifeCycle: LifeCycle) {
+  function handleLog(logsMessage: LogsMessage, logger: Logger, savedCommonContext?: CommonContext) {
+    const messageContext = logsMessage.context
+
+    if (isAuthorized(logsMessage.status, HandlerType.console, logger)) {
+      display.log(`${logsMessage.status}: ${logsMessage.message}`, combine(logger.getContext(), messageContext))
+    }
+
+    lifeCycle.notify(LifeCycleEventType.RAW_LOG_COLLECTED, {
+      rawLog: {
+        message: logsMessage.message,
+        status: logsMessage.status,
+        origin: isExperimentalFeatureEnabled('forward-logs') ? ErrorSource.LOGGER : undefined,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏related to previous PRs, should we rename `ErrorSource` since it does not seem to be related to error anymore?

Reply:
Indeed. I'm wondering if the issue is only on naming. Now that it serve `log origin` and `rum error source`, I feel like it starts to be different concepts. We already have divergence:
```
  AGENT: 'agent',
  CONSOLE: 'console',
  CUSTOM: 'custom', // specific RUM
  LOGGER: 'logger', // specific Logs
  NETWORK: 'network', // specific Logs
  SOURCE: 'source',
  REPORT: 'report',
```
IMO, there is two options:

- keep a single list in coe packahe with a better name like: `apiSource` 

- have two different list:

`logOrigins` in Logs package
`errorSource` in RUM core package

Wdyt?

Reply:
lgtm, we should probably postpone that to a next PR though

Reply:
What option exactly look good to you? :)

Reply:
😅 two different lists

Reply:
Yes I'll do it in a following PR